spec/index.html

```diff
@@ -684,12 +687,18 @@ <h2>Literals</h2>
         language tag MUST be well-formed according to
         <a data-cite="bcp47#section-2.2.9">section 2.2.9</a>
         of [[!BCP47]].</li>
+      <li>if and only if the <a>datatype IRI</a> is
+        <code>http://www.w3.org/1999/02/22-rdf-syntax-ns#langString</code>,
+        <dfn>base direction</dfn> MUST be empty, `ltr` or `rtl`.</li>
     </ul>
 
     <p>A literal is a <dfn>language-tagged string</dfn> if the third element
       is present. Lexical representations of language tags MAY be converted
       to lower case. The value space of language tags is always in lower
-      case.</p>
+      case.
```

Comment:
This also suggests standard formatting in BCP47 and update the sentence about language-tag values.

```suggestion
      to lower case,
      or formatted according to  the recommendation for formatting language tags
      as described in <a data-cite="BCP47#section-2.1.1">section 2.1.1</a> of [[BCP47]].
      The value of language tags is always treated as being in lower case
```